openassessment/templates/openassessmentblock/student_training/student_training.html

```diff
@@ -30,7 +30,8 @@ <h2 class="step__title">
           <span class="step__status__label">{% trans "This step's status" %}:</span>
           <span class="step__status__value">
               <span class="copy">{% trans "In Progress" %}
-              (<span class="step__status__value--completed">{{ training_num_completed }}</span> of
+              (<span class="step__status__value--completed">{{ training_num_completed }}</span>
+                {% trans "of" %}
```

Comment:
OK, I should have figured you were on top of this. I'll hold off on this PR until you have the more sophisticated solution ready.